CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ##### Ghavoran
 
 - Changed: The connection of EMMI Zone Exit Southeast and EMMI Zone Exit West is now a proper door. This enables it to now be shuffled in door lock rando.
+- Changed: Going backwards through the Eyedoor now requires having first destroyed it, Flash Shift and Intermediate Movement, or being able to tank the damage.
```

Comment:
Either specify the eyedoor or change it to say `an eyedoor` so it's more general for other ones

Reply:
But there's only one eyedoor in Ghavoran?

Reply:
Oh I forgot it's in its own section. Ignore.